docs/Homebrew-and-Python.md

```diff
@@ -9,7 +9,26 @@ Homebrew provides formulae to brew 3.x and a more up-to-date Python 2.7.x.
 **Important:** If you choose to install a Python which isn't either of these two (system Python or brewed Python), the Homebrew team cannot support any breakage that may occur.
 
 ## Python 3.x or Python 2.x
-Homebrew provides one formula for Python 2.7.x and another for Python 3.x. The executables are organized as follows so that Python 2 and Python 3 can both be installed without conflict:
+Homebrew provides one formula for Python 2.7.x and another for Python 3.x. 
```

Comment:
I think this communicates all the same information more concisely. I'd rather not duplicate the `caveats`. 

```markdown
Homebrew provides one formula for Python 3.x (`python`) and another for Python 2.7.x (`python@2`) 
```